pkg/config/config.go

```diff
@@ -229,10 +229,11 @@ type NodeSelectorConfig struct {
 }
 
 type SignalRelayConfig struct {
-	Enabled     bool          `yaml:"enabled"`
-	MaxAttempts int           `yaml:"max_attempts"`
-	Timeout     time.Duration `yaml:"timeout"`
-	Backoff     time.Duration `yaml:"backoff"`
+	Enabled          bool          `yaml:"enabled"`
+	MaxAttempts      int           `yaml:"max_attempts"`
+	Timeout          time.Duration `yaml:"timeout"`
+	Backoff          time.Duration `yaml:"backoff"`
+	StreamBufferSize int           `yaml:"stream_buffer_size"`
```

Comment:
is this a buffer per client connection? or a buffer shared across all clients?

Reply:
it's the server's buffer on the media node shared between all signal clients